docs/components/Wrapper.jsx

```diff
@@ -17,11 +17,16 @@ const styles = {
     padding: '1rem'
   }
 }
+
+const themes = ['normal', 'inverted']
```

Comment:
I don't know if this is the way to go to screenshot the inverted theme. 

Can't we imagine having the screenshot script (https://github.com/cozy/cozy-ui/blob/master/scripts/screenshots.js#L97) to click on the "inverted" button and then make the screenshots? 

Also, having within the screenshot's name the value of the theme can be useful.

Reply:
An other way to handle that can be: 

- run the script as actually 
- set the localStorage item `theme` to inverted
- re-run the screenshots

Reply:
J'ai un peu modifié pour faire ce que je voulais faire initialement : par défaut afficher également le thème inversé quand on dev un composant. Du coup maintenant est-ce que ça a plus de sens pour toi de garder le isTesting dans le wrapper ?

Par ailleurs j'ai quand même pris le temps de refacto et comprendre le fonctionnement du script screenshot en question au cas où...

Reply:
Ok je suis parti sur ton idée 👍